ddtrace/llmobs/_evaluators/ragas/faithfulness.py

```diff
@@ -53,21 +47,7 @@ def _get_ml_app_for_ragas_trace(span_event: dict) -> str:
     return "{}-{}".format(RAGAS_ML_APP_PREFIX, ml_app)
```

Comment:
Is this MiniRagas class still necessary here?

Reply:
Nope, I forgot to delete this in my refactor